src/sass/toggle-button-group/toggle-button-group.scss

```diff
@@ -1,7 +1,62 @@
 @import "../variables/variables";
 
+// Recursive Mixin to build repetative class delcarations for explicit dev overrides for various breakpoints
+@mixin toggle-button-grid-templates($breakpoint, $repeatTo, $i: 1) {
+    // $attr: 'data-columns-@{breakpoint}="${i}"';
+    // $selector: '.toggle-button-group[${attr}] ul';
+
+    @if $i <= $repeatTo {
+        .toggle-button-group[data-columns-#{$breakpoint}="#{$i}"] ul {
+            grid-template-columns: repeat($i, 1fr);
+        }
+
+        @include toggle-button-grid-templates($breakpoint, $repeatTo, ($i + 1));
+    }
+}
```

Comment:
Should we move this a common place under `mixins` folder?

Reply:
I thought about that, but it's so specific to the toggle button groups and I can't think of any other use case for this be a general utility.